_data/projects/IcebergAccountingSuite.yml

```
@@ -0,0 +1,19 @@
+name: Iceberg Accounting Suite
+desc: Iceberg Accounting Suite is intended to be a full service accounting suite capable of running small business applications such as invoicing, inventory, a cash drawer, card readers and POS compatability, website connectivity, and more. Written in Python using PySimpleGUI, Iceberg currently allows users to create an encrypted database and journal transactions against a set of accounts with vendor tracking.
+site: https://github.com/josephmbasile/IcebergAccountingSuite
+tags:
+  - accounting
+  - bookkeeping
+  - pos
+  - kms
+  - card
+  - reader
+  - register
+  - inventory
+  - vendor
+  - customer
+  - accounts
+  - books
+upforgrabs:
+  name: help wanted
+  link: https://github.com/josephmbasile/IcebergAccountingSuite/issues
```

:art: keeping this URL in sync with the label name

```suggestion
  link: https://github.com/josephmbasile/IcebergAccountingSuite/labels/help%20wanted
```